src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const pricePerDay = 40;
+  const totalAmount = pricePerDay * days;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    return totalAmount - longTermDiscount;
+  };
+
```

Comment:
Moved the variable declarations (pricePerDay, totalAmount, longTerm, shortTerm, longTermDiscount, shortTermDiscount) inside the function scope, as they are only used within the function.